docs/docs/contribute/getting-started.md

```diff
@@ -14,109 +14,193 @@ license: CC-BY-SA-4.0
+        rustup +nightly target add x86_64-unknown-linux-gnu
+        rustup default nightly-x86_64-unknown-linux-gnu
+        ```
 
 To add support for building to web browsers, input the following command:
-	```
+    ```
     rustup +nightly target add wasm32-unknown-unknown
-	```
+    ```
 
 Use the `rustup update` command to update your installation. Since we are using the nightly release, you are encouraged to update often.
-	```
+    ```
     rustup update
-	```
- 
-<!-- For Linux / WSL
+    ```
 
-Only 2 commands are required to set up the Rust nightly build:
 
-rustup install nightly
-rustup default nightly
+##  Git & GitHub
 
--->
+Git is a system used for managing and tracking changes in your code. It is also known as a Version Control System (VCS). It makes it easier for collaborators to work together, and allows you to access our projects too!
 
-##  Git & GitHub ##
+=== "Windows"
+    Install Git [here](https://git-scm.com/download). The installation settings can be left unchanged.
 
-Git is a system used for managing and tracking changes in your code. It is also known as a Version Control System (VCS). It makes it easier for collaborators to work together, and allows you to access our projects too!
+    Alternatively, you can also use chocolatey:
+    ```cmd
+    choco install -y git
+    ```
 
-Install Git [here](https://git-scm.com/download). The installation settings can be left unchanged.
+=== "Linux"
+    ```bash
+    sudo apt install git
+    ```
 
 Although Git is powerful, it can be daunting to use on its own. Instead, we use GitHub, a service which employs Git’s version control with its own to make project collaboration much easier. 
 
 We recommend that you use a [Git GUI Client](https://git-scm.com/downloads/guis) to facilitate your work by pruning off the manual input of code and making the version control process more visual.
 
-We will be using [GitHub Desktop](https://desktop.github.com/) for the tutorials. You can install it with the default settings, and create a GitHub account when prompted if you do not have one.
+=== "Windows"
+
+    We will be using [GitHub Desktop](https://desktop.github.com/) for the tutorials. You can install it with the default settings, and create a GitHub account when prompted if you do not have one.
+    
+    Alternatively, you can also use chocolatey:
+    ```cmd
+    choco install -y github-desktop
+    ```
+
+    !!! note
+        Even though we recommend GitHub Desktop, feel free to use any GUI client of your choice!
+
+
+## Code Editor
+
+A code editor allows you to work with code more easily. We will be using Visual Studio Code (VS Code) for this tutorial.
+
+=== "Windows"
+
+    You can download it [here](https://code.visualstudio.com/download), and install it with the default settings.
+
+    Alternatively, you can also use chocolatey:
+    ```cmd
+    choco install -y vscode
+    ```
+
+=== "Linux"
+
+    The easiest is to install the snap package.
```

Comment:
Some mention of what the code is meant to do would be good. 

`You can install VS Code with the snap package. Use the command:`